.ci/setup.sh

```diff
@@ -121,11 +121,10 @@ else  # Linux
         fi
     fi
     if [[ $SETUP_CONDA != "false" ]]; then
-        if [[ $ARCH == "x86_64" ]]; then
-            curl -sL -o conda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-        else
-            curl -sL -o conda.sh https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-${ARCH}.sh
-        fi
+        curl \
+            -sL \
+            -o conda.sh \
```

Comment:
```suggestion
            -o mambaforge.sh \
```
and here
https://github.com/microsoft/LightGBM/blob/e6a2f7162cf0bae4cbad6223c2700d74f0d67c4b/.ci/setup.sh#L134

Reply:
updated this in https://github.com/microsoft/LightGBM/pull/4953/commits/bc750fbd3523bd6250f35fd4538c0a94c8c03f4b instead of accepting in the browser, to batch all changes together and avoid duplicate CI runs.